contrib/localnet/docker-compose.yml

```diff
@@ -233,9 +238,9 @@ services:
         ipv4_address: 172.20.0.2
     entrypoint: ["/work/start-zetae2e.sh", "local"]
     environment:
-      - LOCALNET_MODE=${LOCALNET_MODE}
-      - E2E_ARGS=${E2E_ARGS}
-      - UPGRADE_HEIGHT=${UPGRADE_HEIGHT}
+      - LOCALNET_MODE=${LOCALNET_MODE-}
+      - E2E_ARGS=${E2E_ARGS-}
+      - UPGRADE_HEIGHT=${UPGRADE_HEIGHT-}
```

Comment:
Are these missing timespans?

Reply:
docker compose has some specific variable interpolation syntax: https://docs.docker.com/compose/environment-variables/variable-interpolation/#interpolation-syntax

so `${VAR-}` will expand to empty string if `VAR` is unset